index.html

```diff
@@ -1,92 +1,111 @@
+                <a href="https://www.youtube.com/watch?v=6v577ho-mw8&list=PLH9fqbYeMD3MwDt_wozHviB-Naamd-J61&index=4"><img src="images/thumb2.webp" alt="thumbnail pic" width="100%">
+                </a>
 
-    For now, the test suite only works in Chrome!
-    Please read the README below in the JS Editor before beginning.
-    Feel free to delete this message once you have read it.
-    Good luck and Happy Coding!
+            </div>
+            
+            <div>
+                <p>
+                    5th video
+                </p>
+                <a href="https://www.youtube.com/watch?v=0khJEvtMyRA&list=PLH9fqbYeMD3MwDt_wozHviB-Naamd-J61&index=5"><img src="images/thumb3.webp" alt="thumbnail pic" width="100%">
+                </a>
+               
+            </div>
+            
+            <div>
+                <p>
+                    6th video
+                </p>
+                <a href="https://www.youtube.com/watch?v=0ma8iI_AblM&list=PLH9fqbYeMD3MwDt_wozHviB-Naamd-J61&index=6"><img src="images/thumb4.webp" alt="thumbnail pic" width="100%">
+                </a>
+            </div>
+            
+        
+        </div>
+        <div>
+            <a href="https://www.microverse.org/" target=”_blank”><img src="images/advertisment.png" alt="advertising pic" height="auto" width="100%"></a></div>
+    </div>
+    
 
-    - The freeCodeCamp Team
-
-    -->
-    <!DOCTYPE html>
-<html lang="en">
-<head>
-	<meta charset="UTF-8">
-	<meta http-equiv="X-UA-Compatible" content="IE=edge">
-	<meta name="viewport" content="width=device-width, initial-scale=1.0">
-	<title>Document</title>
-</head>
-<body>
-	<main id="main">
-	<h1 id="title">Dr. Norman Borlaug</h1>
-	<p>The man who saved a billion lives</p>
-  <div id="img-div">
-	<img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt="Dr. Norman Borlaug">
-<p id="img-caption">	Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</p>
-</div>
-<br>
-<br>
-<br>
-<div id="tribute-info">
-	<h2>Here's a time line of Dr. Borlaug's life:</h2>
-	<br>
-	<br>
-	<br>
-	<ul>
-		<li><strong>1914</strong>- Born in Cresco, Iowa</li>
-		<li><strong>1933</strong>- Leaves his family's farm to attend the University of Minnesota, thanks to a Depression era program known as the "National Youth Administration"</li>
-		<li><strong>1935</strong> - Has to stop school and save up more money. Works in the Civilian Conservation Corps, helping starving Americans. "I saw how food changed them", he said. "All of this left scars on me."</li>
-		<li><strong>1937</strong>- Finishes university and takes a job in the US Forestry Service</li>
-		<li><strong>1938</strong>- Marries wife of 69 years Margret Gibson. Gets laid off due to budget cuts. Inspired by Elvin Charles Stakman, he returns to school study under Stakman, who teaches him about breeding pest-resistent plants.</li>
```

Comment:
Please Add all paragraph

Reply:
- Please always explain why the requested changes are required, so the student gets a better understanding or what is needed to change on his/her code.